internal/pkg/cli/svc_deploy.go

```diff
@@ -399,11 +420,21 @@ func (o *deploySvcOpts) runtimeConfig(addonsURL string) (*stack.RuntimeConfig, e
 	if err != nil {
 		return nil, err
 	}
+
+	svc, err := o.manifest()
```

Comment:
Can we move this call inside `o.snsTopics()` and refactor the `snsTopics` function to not use an input service and instead rely on the `o.name` variable?